cves/CVE-2013-4316.yml

```diff
@@ -45,7 +45,13 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Struts 2 introduced a new feature called Dynamic Method Invokation (DMI). This
+  allowed developers to call specific public methods by passing them through the
+  url after an exclamation point. This opened up the possibilty for users to call
+  other methods by abusing this feature. DMI is defaulted to on for Struts versions
+  2.0.0 through 2.3.15.1. Users were not warned about the potential security risks
+  of leaving this enabled.
```

Comment:
One simple spelling error - I believe the spelling is "Invocation", and that is how it is spelled on the Struts website. Also, do you have more specific information about how this was abused? Did attackers need to use crazy attack vectors to invoke methods that weren't supposed to be called? That information would be interesting to know, but it's not critical.

Reply:
This spelling appears twice more in the file.

Reply:
I have fixed the spelling as well as added in some examples as to how this could lead to exploits.